app/models/sp_return_log.rb

```diff
@@ -1,3 +1,8 @@
 class SpReturnLog < ApplicationRecord
+  # rubocop:disable Rails/InverseOf
```

Comment:
I added these for convenience when creating these records in specs, I think in prod we'd want to discourage looking up rows in this DB so am leaving out the inverse relations for now